File: backend/src/main/java/com/orange/Crisalis/dto/RequestBodyCreateOrderDTO.java
package com.orange.Crisalis.dto;

import lombok.*;

import java.util.List;


@Getter
@Setter
@Data
@AllArgsConstructor
@NoArgsConstructor
public class RequestBodyCreateOrderDTO {

    private Integer clientId;
    private List<ProductIdAndQuantityDTO> productIdList;


}

File: frontend/src/app/order/request-body-create-order-dto.ts
import { ProductIdAndQuantityDto } from "./product-id-and-quantity-dto";

export class RequestBodyCreateOrderDto {
  clientId: number;
  productIdList: ProductIdAndQuantityDto[];

  constructor(clientId: number, productIdList: ProductIdAndQuantityDto[]) {
    this.clientId = clientId;
    this.productIdList = productIdList;
  }
}
